.golangci.yml

```diff
@@ -240,14 +240,12 @@ linters:
   enable:
     - gofmt
     # - govet
-    # - errcheck
+    - errcheck
     # - staticcheck
     # - unused
     # - gosimple
     # - structcheck
-    - varcheck
     - ineffassign
-    # - deadcode
```

Comment:
Why is this one deleted? Do we want to reenable it at some point in the future? FYI, when enabled, this check fails because there are two functions (`includeConfiguredNssaiCheck` and `refreshSmfProfiles`) that are unused.

Reply:
This linter has also been abandonned by its upstream, last seeing an update 7 years ago: https://github.com/remyoudompheng/go-misc/tree/master/deadcode.

The `unused` linter will probably cover the same use cases and is maintained.